app/actors/curation_concerns/actors/add_to_collection_actor.rb

```diff
@@ -22,8 +22,11 @@ def add_to_collections(new_collection_ids)
           # remove from old collections
           (curation_concern.in_collection_ids - new_collection_ids).each do |old_id|
             collection = ::Collection.find(old_id)
-            collection.members.delete(curation_concern)
-            collection.save
+            # don't remove from collections current user doesn't have edit permissions for
+            if collection.edit_users.include? user.email
```

Comment:
Same question as https://github.com/projecthydra/curation_concerns/pull/1132/files#r95906550

Wouldn't you need a way for admins to delete also?